corehq/apps/smsforms/tests/test_session_synchronization.py

```diff
@@ -0,0 +1,43 @@
+from collections import namedtuple
+from uuid import uuid4
+
+from corehq.apps.smsforms.models import XFormsSessionSynchronization, RunningSessionInfo
+
+
+def test():
+    phone_number_a = _clean_up_number('15555555555')
+    phone_number_b = _clean_up_number('15555555554')
+    session_a_1 = FakeSession(session_id=str(uuid4()), phone_number=phone_number_a, connection_id='Alpha')
+    session_a_2 = FakeSession(session_id=str(uuid4()), phone_number=phone_number_a, connection_id='Beta')
+    session_b_1 = FakeSession(session_id=str(uuid4()), phone_number=phone_number_b, connection_id='Kappa')
+
+    # Nothing set yet, so it can be claimed
```

Comment:
Thanks for the comments, they're a great help.

Reply:
You're welcome :) They're actually helpful even just for me when I'm rereading them